pages/common/hub.md

```diff
@@ -11,6 +11,14 @@
 
 `hub clone {{github_username}}/{{repo_name}}`
 
+- Fork repository cloned from another user (creates a remote named after your GitHub username):
```

Comment:
This is a bit confusing, especially since fork and clone have closely related meaning. I'm wondering if we could rewords and/or mention a specific example of what would happen (in fact, I assume the command would interactively ask for information -- maybe we could mention what info the user would need to provide).